projects/patches/openvdb-remove-lib-prefix-static.patch

```diff
@@ -0,0 +1,24 @@
+From 1f41a0d72b6f51f117888adece18a61a2a11c6e7 Mon Sep 17 00:00:00 2001
+From: Mathieu Westphal <mathieu.westphal@kitware.com>
+Date: Mon, 1 Jan 2024 14:59:26 +0100
+Subject: [PATCH] openvdb remove lib prefix static
+
+---
+ openvdb/openvdb/CMakeLists.txt | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/openvdb/openvdb/CMakeLists.txt b/openvdb/openvdb/CMakeLists.txt
+index 33d4e5c..2c16443 100644
+--- a/openvdb/openvdb/CMakeLists.txt
++++ b/openvdb/openvdb/CMakeLists.txt
+@@ -678,7 +678,6 @@ if(OPENVDB_CORE_STATIC)
+       target_compile_options(openvdb_static PUBLIC
+         "$<$<COMPILE_LANG_AND_ID:CXX,MSVC>:/MT$<$<CONFIG:Debug>:d>>")
+     endif()
+-    set_target_properties(openvdb_static PROPERTIES PREFIX "lib")
```

Comment:
do we care about this patch?

Reply:
we care a lot. Without it, the FindOpenVDB.cmake doesnt work on Windows with static openvdb.

Clearly untested code.

Reply:
ok ^^